dss/api/subscriptions.py

```diff
@@ -180,3 +192,6 @@ def _register_subscription(es_client: Elasticsearch, uuid: str, json_request_bod
                            id=uuid,
                            body=json_request_body,
                            refresh=True)
+
+def _get_indexes_by_alias(es_client: Elasticsearch, alias_name: str):
+    return [indexes['i'] for indexes in es_client.cat.aliases(name=alias_name, h=['i'], format='json')]
```

Comment:
`list(es_client.indices.get_alias('foo').keys())`

Reply:
Yes, that works unless the alias is not defined, in which case either an exception is thrown (default), or an error tuple is returned (if "ignore=404" is also passed in). I also don't like using the 'cat' API. Instead, we may want to wrap the snippet you provided in an easy-to-use convenience function that does what we need it to, instead of using 'cat'.

Reply:
I'd prefer

```
    try:
        return list(es_client.indices.get_alias(alias_name).keys())
    except NotFoundError:
        return []
```

Reply:
Yup, looks good to me. Thanks!

Reply:
fixed